assets/queries/dockerCompose/seccomp_disabled/metadata.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "404fde2c-bc4b-4371-9747-7054132ac953",
+  "queryName": "Seccomp Disabled",
```

```suggestion
  "queryName": "Default Seccomp Profile Disabled",
```

Please, update the query folder name